File: testprojects/rel_scala_2_4_project/pipelines/Pipeline_Scala2-11_Spark2-4/code/src/main/scala/io/prophecy/pipelines/pipeline_scala211_spark24/graph/Reformat_1.scala
package io.prophecy.pipelines.pipeline_scala211_spark24.graph

import io.prophecy.libs._
import io.prophecy.pipelines.pipeline_scala211_spark24.config.ConfigStore._
import io.prophecy.pipelines.pipeline_scala211_spark24.config.Context
import io.prophecy.pipelines.pipeline_scala211_spark24.udfs.UDFs._
import io.prophecy.pipelines.pipeline_scala211_spark24.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }

File: testprojects/master_python_mainone/pipelines/PYTHON_DEP_MANAGEMENT_ALL/code/job/graph/SubGraph_2/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from job.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(col("`c- short`"), col("p_timestamp"))
